assets/js/customizer.js

```diff
@@ -7,108 +7,124 @@
 				addItemButton = container.querySelector(".add-featured-item");
 			const { __, _x, _n, _nx } = wp.i18n;
 			let featuredArea,
-				itemObjects = new Map(),
-				timer,
+				settings_timer,
 				search_timer,
 				timer_ms = 500;
 
 			class ListItem {
-				constructor(post) {
-					this.key = post.ID;
+				constructor(post, parent) {
+					this.id = post.id;
 					this.postData = post;
-					this.featured_area = control.id;
-					this.element_id = "item_" + post.ID;
+					this.parent = parent;
+					this.element = null;
+					this.element_id = control.id + "_item_" + post.id;
```

Comment:
I think we should use this.id instead of post.id here, just to be consistent. What do you say @redundans